scalding-core/src/test/scala/com/twitter/scalding/StringUtilityTest.scala

```diff
@@ -0,0 +1,95 @@
+package com.twitter.scalding
+
+import org.scalatest.{ Matchers, WordSpec }
+
+class StringUtilityTest extends WordSpec with Matchers {
```

Comment:
@isnotinvain Now that I dropped the part to match java's split, I'm dropping this test for comparing fastSplit and java's regex split.